build.gradle

```diff
@@ -18,9 +18,9 @@
 
 buildscript {
     ext {
-        hiltVersion = "2.38.1"
+        hiltVersion = '2.40.1'
```

Comment:
Please change to double quotes for consistency.

Reply:
Done